### PR TITLE
Who-tests-what nightly jenkins build

### PR DESCRIPTION
**Pull request by cpennington**

### CODE REVIEW

File: platform/jobs/pipelines/edxPlatformPipelinePrWTW.groovy [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+}
+
+Map publicBokchoyJobConfig = [
+    open: true,
+    jobName: 'edx-platform-bokchoy-pipeline-pr',
+    repoName: 'edx-platform',
+    whitelistBranchRegex: /^((?!open-release\/).)*$/,
+    context: 'jenkins/bokchoy-wtw',
+    onlyTriggerPhrase: true,
+    triggerPhrase: /.*jenkins\W+run\W+bokchoy\W+wtw.*/,
```
Comment (estute) [inline platform/jobs/pipelines/edxPlatformPipelinePrWTW.groovy]: I believe these will also match to the regular expressions in the normal PR job trigger phrases. You should make them more unique, like "jenkins run wtw bokchoy" or similar.

File: platform/jobs/pipelines/edxPlatformPipelinePrWTW.groovy [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+Map publicPythonJobConfig = [
+    open: true,
+    jobName: 'edx-platform-python-pipeline-pr',
+    repoName: 'edx-platform',
+    whitelistBranchRegex: /^((?!open-release\/).)*$/,
+    context: 'jenkins/python-wtw',
+    onlyTriggerPhrase: true,
+    triggerPhrase: /.*jenkins\W+run\W+python\W+wtw.*/,
+    jenkinsFileDir: 'scripts/Jenkinsfiles',
+    jenkinsFileName: 'python'
```
Comment (estute) [inline platform/jobs/pipelines/edxPlatformPipelinePrWTW.groovy]: ^ are you still thinking about using the existing Jenkinsfiles and just adding to them or did you want to create a new one?

File: platform/jobs/pipelines/edxPlatformPipelinePrWTW.groovy (hunk 1 above)
Comment (cpennington) [reply platform/jobs/pipelines/edxPlatformPipelinePrWTW.groovy]: Good call.

File: platform/jobs/pipelines/edxPlatformPipelinePrWTW.groovy (hunk 2 above)
Comment (cpennington) [reply platform/jobs/pipelines/edxPlatformPipelinePrWTW.groovy]: I think I can use the existing jenkinsfiles and just modify the arguments passed to paver/pytest/coverage based on the environment variables.

Comment (estute) [reply platform/jobs/pipelines/edxPlatformPipelinePrWTW.groovy]: got it

**Pull request merged**